### ISSUE TITLE
The default template should inspect the current folder to offer smarter defaults

### ISSUE THREAD

**Issue by orta** [danger/danger#90]
We should have templates that adapt to the repo `init` was run in:

- [ ] A Gem
- [ ] An npm module
- [ ] Rails apps
- [ ] Sinatra projects
- [ ] Node apps
- [ ] React specific apps
- [ ] A Pod / SPM module
- [ ] iOS App

Note: This is a living document and I'll keep updating with ideas here.

**Comment by orta** [comment]
Got some ideas from @jamtur01 today - thanks

**Comment by marcelofabri** [comment]
I think we should be careful about where to draw the line. IMO some of these ideas are not about "Pull Request etiquette" but about linting the project. They could be implemented as plugins, though.

**Comment by orta** [comment]
Agree, linting shouldn't but changing `*.[pod|gem]specs` probably should. Interesting, still worth some more thought.

**Comment by orta** [comment]
in https://github.com/artsy/artsy.github.io/pull/217 - I wondered about a thing specific for blogging where Danger can check prose and links

**Comment by orta** [comment]
Swift package manager: https://github.com/VeniceX/HTTPServer/pull/6/files#diff-16417461de376720abfe05ef4cb7d73aR4

**Comment by orta** [comment]
A gem: https://github.com/realm/jazzy/pull/572/files#diff-16417461de376720abfe05ef4cb7d73aR1
